### ISSUE TITLE
Error: Cannot read properties of null (reading 'useState')

### ISSUE THREAD

**Issue by UsamaKhan** [tremorlabs/tremor-npm#387]
When trying to use Tremor, I get the following error:

![image](https://user-images.githubusercontent.com/3194581/230143098-f89a23d6-b8f8-49ae-9dc2-339a326947e6.png)

Here is my source code:
```tsx
import { PrismaClient } from "@prisma/client";
import {
  Card,
  Title,
  Text,
  Flex,
  Table,
  TableRow,
  TableCell,
  TableHead,
  TableHeaderCell,
  TableBody,
  Badge,
  Button,
} from "@tremor/react";

export default async function page() {
  const prisma = new PrismaClient();
  const users = await prisma.user.findMany();

  return (
    <main>
      <Card>
        <Flex justifyContent="start" className="space-x-2">
          <Title>Users</Title>
          <Badge color="gray">{users.length}</Badge>
        </Flex>
        <Text className="mt-2">Overview of all site users</Text>

        <Table className="mt-6">
          <TableHead>
            <TableRow>
              <TableHeaderCell />
              <TableHeaderCell>Name</TableHeaderCell>
              <TableHeaderCell>Username</TableHeaderCell>
              <TableHeaderCell>Email</TableHeaderCell>
              <TableHeaderCell>Role</TableHeaderCell>
              <TableHeaderCell />
            </TableRow>
          </TableHead>

          <TableBody>
            {users.map((user) => (
              <TableRow key={user.username}>
                <TableCell>{user.image}</TableCell>
                <TableCell>{user.fname} {user.lname}</TableCell>
                <TableCell>{user.username}</TableCell>
                <TableCell>{user.email}</TableCell>
                <TableCell>{user.role}</TableCell>
                <TableCell>
                  <Button size="xs" variant="secondary">
                    Edit
                  </Button>
                </TableCell>
              </TableRow>
            ))}
          </TableBody>
        </Table>
      </Card>
    </main>
  )
}
```

**Comment by brycepedroza** [comment]
I assume you're using Next.js 13. This issue is called out in the [Getting Started doc](https://www.tremor.so/docs/getting-started/installation)
> Since we have not fully migrated to Next.js 13 yet, if you are using the `app` directory introduced in Next.js 13, wrap your tremor components in another component by using the `"use client"` directive.

You can find how to add that in the Next.js documentation [here](https://beta.nextjs.org/docs/rendering/server-and-client-components#convention)

TL;DR:
add `'use client';` to the top of your source code.

Cheers!

**Comment by severinlandolt** [comment]
Hey @UsamaKhan, did @brycepedroza's (thank you btw!) solution work for you?
